README.md

```diff
@@ -165,7 +165,8 @@ maintained by [Parity Technologies](https://www.parity.io/). Source code availab
 - [Fearless Utils Android](https://github.com/soramitsu/fearless-utils-Android) - Android Substrate tools.
 
 ## Tools
-
+- [fudge](https://github.com/centrifuge/fudge) - Core lib for accessing and (arbitrarily) manipulating substrate databases without a substrate-node. 
+  Allows to build and import blocks locally using the standard build and import mechanisms. 
```

Comment:
All awesome list items can only be one sentence and one line.

Reply:
@NukeManDan updated.